libs/blocks/marquee/marquee.css

```diff
@@ -497,6 +492,14 @@
     justify-content: flex-start;
   }
 
+  html[dir="rtl"] .marquee.support-rtl.split .foreground.container {
+    flex-direction: row;
+  }
+
+  html[dir="rtl"] .marquee.support-rtl.split.row-reversed .foreground.container {
+    flex-direction: row-reverse;
+  }
```

Comment:
I think this may have leaked from another story regarding split marquees on RTL, is this supposed to be here? Same question applies to the whole file actually.

Reply:
You are correct, I have reverted this file. I must have been on this branch when i created a new one. Thanks for the catch